### DISCUSSION TITLE
[bug]Calendar startCalendar and endCalendar ignore the disabled dates

### DISCUSSION THREAD

**Discussion by Wolclaws** [fomantic/Fomantic-UI#667] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
# Bug Report

I have two calendars linked with startCalendar and endCalendar if there are some disabled dates between the  first and the second selected dates the calendars work like if disabled dates aren't disabled.

Sorry for my bad english and this is my first time on github I hope I have presented the problem at its best

## Steps to reproduce
1. make two calendars with  startCalendar and endCalendar 
2. disable a date
3. on the first calendar select the day before the disabled date and on the second the day after

## Expected result
 endCalendar should block the selection on any dates after the disabled day
## Actual result
 startCalendar and endCalendar ignore the disabled dates

## Live example

https://jsfiddle.net/jym10Lud/

## Version
2.7

**Comment by exoego** [comment]
@Wolclaws 
To avoid misunderstanding the issue, could you provide a reproducible example by forking https://jsfiddle.net/31d6y7mn

**Comment by Wolclaws** [comment]
Added an example https://jsfiddle.net/jym10Lud/

**Comment by exoego** [comment]
@Wolclaws 
`disabledDates` is a setting to disable specific individual dates e.g. holidays or some special days.
IIUC, it can not be used to disable "days after the specified dates".

Let me confirm that your want to have a new setting to disable "days after the specified date" ?
![image](https://user-images.githubusercontent.com/127635/56207987-09a46480-608b-11e9-8815-ac7cd7807e5c.png)

**Comment by lubber-de** [comment, marked as answer]
@Wolclaws To gain your desired behavior(and described by  @exoego ), you can achieve this by using `maxDate`. Set this to the lowest value of disabledDates
See https://jsfiddle.net/L5w7vtcj/1/

**Comment by Wolclaws** [comment]
Sorry for the delay in replying.

@exoego Not exactly, I think it is more useful a new setting (maybe boolean) to disable "days after a disabled day if this is included in the selected period between two or more calendars linked by startCalendar and endCalendar".

@lubber-de Yours is a good solution, but there is some problem with the "set maxDate" behavior. The console returns this error to me:
...
Calendar: The method you called is not defined.
Array ["set", "maxDate (.......
...

Can you help me?

Here is my code that you can also use as a basis for the setting I was talking about with @exoego 

https://jsfiddle.net/s8j3g46y/4/

**Comment by lubber-de** [comment]
@Wolclaws You have to provide the maxdate as a separate argument
```javascript
$('.end').calendar('set maxDate',maxdate);
```
https://jsfiddle.net/y76oLwf3/

**Comment by Wolclaws** [comment]
@lubber-de Perfect thanks! I had taken an oversight ^^"

**Comment by lubber-de** [comment]
Closing since desired behavior is doable using existing features.
